src/redux/reducers/TrailReducer.js

```diff
@@ -56,6 +56,7 @@ export default (state = initialState, action) => {
     }
     case TRAIL_MODIFIED: {
       const { id, editedFields } = action.data;
+
```

Comment:
Not a big deal, but it's nice to give a quick read-through of "files changed" in your PR's and see if things like this snuck in. All changes should be improvements and have a reason (whether through the feature itself or to fix a code style / linting issue, etc).

Reply:
Gotcha, makes sense. I'll try and be a bit better about that.